docs/product-model.md

```diff
@@ -554,10 +536,62 @@ For instance in an FX spot or forward transaction, the respective units
 of the quantity and price will determine who is paying or receiving each
 currency.
 
-## Financial Product
+## Financial Product {#product}
 
 A financial product is an instrument that is used to transfer financial
-risk between two parties. Financial products are represented in the
+risk between two parties. 
+
+### Asset  {#asset}
+
+The concept of "Asset" has been introduced in the CDM in version 6.0 to improve the modelling of simpler financial products
+and to continue the expansion of the CDM from beyond its heritage focus on over the counter derivatives.
+
+Specifically, the `Asset` data type models something that can be owned and transferred in the financial markets.  As a `choice`
+data type, one and only one of the  attributes must be used:
+
+``` Haskell
+choice Asset:  
+    Cash
+    Commodity
+    DigitalAsset
+    Instrument
+
+```
+
+These attributes are:
+
+* Cash: An asset that consists solely of a monetary holding in a currency.  The only attribute on this
+  data type is an `Identifier` which is a string that should be populated with the `CurrencyCodeEnum` for the currency
```

Comment:
Suggested alternative:
"populated with the currency code (according to the `CurrencyCodeEnum` list)"